src/compiler/parser.ts

```diff
@@ -436,6 +436,13 @@ module ts {
         // attached to the EOF token.
         let parseErrorBeforeNextFinishedNode: boolean = false;
 
+        export const enum StatementFlags {

```

Comment:
Ah, are you suggesting we use the ParsingContext type instead of StatementFlags? I suppose we could, although it has many other flags that we'd never use.